ts-sdk/src/schema/address.ts

```diff
@@ -17,14 +18,30 @@ export type AddressCosmosDisplay = typeof AddressCosmosDisplay.Type
 export const AddressCosmosZkgm = Hex.pipe(S.brand("AddressCosmosZkgm")) // TODO: Hex<Bech32<Hrp, Cosmos.Canonical>>
 export type AddressCosmosZkgm = typeof AddressCosmosZkgm.Type
 
+// export const AddressCosmosDisplayFromCanonical = flow(
+//   Bech32FromAddressCanonicalBytesWithPrefix,
+//   S.compose(AddressCosmosDisplay)
+// )
+
+export const AddressCosmosZkgmFromAddressCanonicalBytesWithPrefix = (
+  prefix: string
+) => pipe(
+  Bech32FromAddressCanonicalBytesWithPrefix(prefix),
+  S.compose(HexFromString),
+  S.compose(AddressCosmosZkgm)
+)
+
```

Comment:
Schema composition to handle transformation to/from canonical to zkgm on cosmos.